src/modules/util.js

```diff
@@ -0,0 +1,56 @@
+import { baseUrl, involvementUrl } from './api.js';
+
+const getData = async () => {
+  const response = await fetch(baseUrl);
+  const data = await response.json();
+  return data;
+};
+
+const getMovie = async (id) => {
+  const movieUrl = `${baseUrl}/${id}`;
+  const response = await fetch(movieUrl);
+  const data = await response.json();
+  return data;
+};
+
+const pushLike = async (id) => {
+  await fetch(`${involvementUrl}likes/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+    }),
+  });
+};
```

Comment:
- [ ] Wonderful job so far on your capstone project. You've really put in a lot. However, I noticed one issue about the `pushLike` function. The page get updated at first click but fail to update the like count on the UI at the subsequent clicks until the page is refreshed. Please, take a look.

[likes-issue.webm](https://user-images.githubusercontent.com/26597432/218180760-3ac1aa5b-177b-40b4-9b5d-4b33cf512e54.webm)
 
You might need to call `getLike()` inside the pushLike to make the like count get updated on the UI. Something like this :point_down: 

```
   const pushLike = async (id) => {
       await fetch(`${involvementUrl}likes/`, {
           method: 'POST',
           headers: {
               'Content-Type': 'application/json',
           },
           body: JSON.stringify({
               item_id: id,
           }),
        });
       getLikes();
   };

```

Reply:
Thanks @Qoosim  For the wonderful review. ✋ 
---

Reply:
You're most welcome :v: